configs/.env.devnet-1

```diff
@@ -0,0 +1,8 @@
+BOOT_NODE_IP_ADDRESS=164.90.155.170
+NETWORK_IDENTIFIER=12D3KooWJ4DNs6Mp6hsJNW45odw6FQ3S7HytybQdBVaPtnd2HDcK
+BOOT_NODE_IP_ADDRESS_2=104.236.193.202
+NETWORK_IDENTIFIER_2=12D3KooWJ4DNs6Mp6hsJNW45odw6FQ3S7HytybQdBVaPtnd2HDcK
+LOG_LEVEL=debug
+NODE_NAME=DEVNET_1_NODE_NAME
+SPEC_PATH=devnet-1
+VERSION=v2.10.0-test-1
```

Comment:
I noticed the version is v2.3.1 in other configs. Shall we align everything?

Reply:
This is the correct version. The reason is that we can change the durations only once in genesis, so we need to have a different image.